.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -49,7 +49,6 @@
 
 ### Does this change update the version of CAPI we're using?
 
-<!-- Changing CAPI versions renders the existing local database files useless -->
```

Comment:
Removing this warning as it's no longer true, we are removing the `User-Agent` header with the CAPI client version in it. Warning added in https://github.com/guardian/frontend/pull/20988